.github/ISSUE_TEMPLATE/003_rfc-proposal.yml

```
@@ -0,0 +1,75 @@
+name: RFC / Proposal
+description: Submit a Proposal (RFC)
+labels: ["RFC"]
+type: RFC
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Introduction
+
+        This section targets end users, TSC members, maintainers and anyone else
+        that might need a quick explanation of your proposed change.
+
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: Problem Description
+      description: Why do we want this change and what problem are we trying to address?
+      placeholder: Explain the problem or limitation this RFC is meant to resolve.
+    validations:
+      required: true
+
+  - type: textarea
+    id: proposed-change-summary
+    attributes:
+      label: Proposed Change (Summary)
+      description: A high-level summary of the proposed change (the 10,000 ft view).
```

I suggest changing this to something less confusing for non-US users:

```suggestion
      description: A high-level summary of the proposed change.
```